pages/story/story_data.js

```diff
@@ -0,0 +1,28 @@
+
+//write functions here to take data and push to story_array
+
+export const story_array = 
+[{
+    num : 1,
+    href: "/story/1", //keep this unique
```

Comment:
I think instead of `href` here we can have it as `slug` where slug will be unique and give an idea what the story is about. like `sectionals-2023` or `ncuc-2022` which will be in the url

Reply:
yeah will be better for the users to input slugs, will change